_includes/cookies/cookie-consent.html

```diff
@@ -0,0 +1,81 @@
+<noscript>
+  <div id="cookie-notice-nojs">
+  <div class="container">
+  <div id="cookie-choice-nojs">
+    <p><strong>Tell us whether you accept cookies</strong></p>
+    <p>We use <a href="/about/your-data">cookies to collect information</a> about how you use this website to improve the functionality.</p>
+  </div>
+  </div>
+  </div>
+</noscript>
+
+<div id="cookie-notice">
+<div class="container">
+
+  <div id="cookie-choice">
+    <p><strong>Tell us whether you accept cookies</strong></p>
+    <p>We use <a href="/about/your-data">cookies to collect information</a> about how you use this website to improve the functionality.</p>
```

Comment:
```suggestion
    <p>We use <a href="/about/your-data#cookies">cookies to collect information</a> about how you use this website to improve the functionality.</p>
```

Think we can add a anchor so bowser takes the user straight to the cookies part of the privacy page